CONTRIBUTING.md

```diff
@@ -11,10 +11,50 @@ requirements and recommendations.
 Before you can contribute, you will need to sign the [Contributor License
 Agreement](https://identity.linuxfoundation.org/projects/cncf).
 
-## TODO
+## How to Contribute
+
+When contributing to semantic conventions, it's important to understand a few
+key, but non-obvious, aspects:
+
+- All attributes, metrics, etc. are formally defined in YAML files under
+  the `model/` directory
+- All descriptions, normative language are defined in the `docs/`
+  directory.
+  - We provide tooling to generate Markdown docuemntation from the formal
+    YAML definitons.  See [Yaml to Markdown](#yaml-to-markdown).
+  - We use Hugo to render [on our website](https://opentelemetry.io/docs/specs/semconv/).
+    You will see `<!--- Hugo front matter used to generate ...` sections
+    in markdown.  See [Hugo frontmatter](#hugo-frontmatter) for details.
+- All changes to existing attributes, metrics, etc. MUST be allowed and
```

Comment:
What does allowed mean in this context?
Does this mean reviewed and approved, or be compliant with our stability guarantees?

Reply:
```suggestion
- All changes to existing attributes, metrics, etc. MUST be allowed as
  per our [stability guarantees](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/versioning-and-stability.md#semantic-conventions-stability) and
```